doc/CHECKINS.md

```diff
@@ -0,0 +1,39 @@
+# Community Check-ins
+
+One of our community values is openness and diversity in thought. For this reason, we have started weekly check-ins run by our contributors (you!). The check-ins are currently opened as issues in the plots2 repository
+ for convenience, but are meant to instigate discussions about any and all repos / topics related to Public Lab. You can always find the most recent check-in pinned at the top of the [issues](https://github.com/publiclab/plots2/issues) page.
+
+New contributors are encouraged to get engaged :).
+
+Below we provide you with information for getting involved and making your own post. These are just guidelines - we encourage you to get creative and lead the check-in with whatever words you would like 💫.
+
+## Rundown 
+
+- First things first: our community aspires to be a respectful place. Please read and abide by our [Code of Conduct](https://publiclab.org/conduct).
+
+- Check-ins are typically opened as issues anytime on Sunday or Monday.
+
+- Volunteer to open the next check-in by leaving a comment on the current one.
+
+-  When you open a check-in please use our [Gitter](https://gitter.im/publiclab/publiclab) chatroom to
+notify everyone by referencing `@/all` and linking the post. Include a brief message for new contributors so that they know what it is. ex:
+
+"<em>Hey! Please see our weekly check-in \<here\>. Everyone is welcome to contribute and we encourage new users to get involved</em>."
+
+- Check-ins have a <em>theme</em>. Select any theme that interests you to guide the general flow of responces from the communtiy. 
+
+- Add anything else you'd like. 
+
+  - Discovered a new technology you'd like to share? 
+
+  - Found an interesting article about open source? 
+
+  - Have any questions? 
+
+  - Troubleshot any issues that the rest of the community can learn from?
```

Comment:
A small change required here: *Troubleshoot

Reply:
It's not a typo.

Reply:
Ah I see, it's correct indeed! Apologies @gauravano, I didn't read that sentence completely. :+1: